.vib/opensearch/goss/goss.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+file:
+/opt/bitnami/opensearch/config/opensearch.yml:
+  mode: "0644"
+  filetype: file
+  contains:
+   - /port.*{{ .Vars.containerPorts.transport }}/
+   - /allow_nmap:.*false/
+  exists: true
+/bitnami/opensearch/data:
```

Comment:
Missing the capabilities test here

Reply:
[c679021](https://github.com/bitnami/charts/pull/17509/commits/c679021d28b5b3abecff3f5354911a3b9a1ad51b)